src/api/java/baritone/api/pathing/goals/GoalComposite.java

```diff
@@ -57,6 +57,16 @@ public double heuristic(int x, int y, int z) {
         return min;
     }
 
+    @Override
+    public double heuristic() {
+        double min = Double.MAX_VALUE;
+        for (Goal g: goals) {
```

Comment:
Plz run through an autoformatter to baritone style

Reply:
Any recommendations for a linux command line formatter that can conform to baritone style?
And if not, maybe some info on what "baritone style" is?

Reply:
uh it's basically the intellij autoformatter? iirc theres very few changes and you won't run into any of them. you can test if you have it right by doing reformat on all the code and checking if `git` says any files have been changed